Pagination/Tests/MongoTrait/FilterTypeStrategy/Strategies/ReferenceFilterStrategyTest.php

```diff
@@ -97,15 +97,19 @@ public function testGenerateFilterWithNoPaginationRepsitory()
     }
 
     /**
-     * @param string $columnsTree
-     * @param array  $expectedFilter
+     * @param string   $columnsTree
+     * @param int|null $countOrFilter
      *
      * @dataProvider provideGenerateFilter
      */
-    public function testGenerateFilter($columnsTree, $expectedFilter)
+    public function testGenerateFilter($columnsTree, $countOrFilter = null)
     {
         $filter = $this->strategy->generateFilter($columnsTree, $this->value, $this->documentName);
-        $this->assertEquals($expectedFilter, $filter);
+        if (null === $countOrFilter) {
```

Comment:
null indicates that there is no filter created.  not that there is a filter with no correspondence